### PR TITLE
Register `wporg_workshop_type` taxonomy

### LINKED ISSUE #259: Add a new `wporg_workshop_type` taxonomy
As per this comment: https://github.com/WordPress/learn/issues/252#issuecomment-952415901

Please add a new taxonomy named `wporg_workshop_type` to the `workshop` post type.

### PR DESCRIPTION
**Pull request by hlashbrooke**
Registers new `wporg_workshop_type` taxonomy.

Fixes #259

### CODE REVIEW

File: wp-content/plugins/wporg-learn/inc/taxonomy.php [hunk 1]
```diff
@@ -429,6 +430,51 @@ function register_workshop_topic() {
...
+		'search_items'               => __( 'Search Types', 'wporg-learn' ),
+		'not_found'                  => __( 'No Type Found', 'wporg-learn' ),
+		'no_terms'                   => __( 'No Type ', 'wporg-learn' ),
+		'items_list'                 => __( 'Type list', 'wporg-learn' ),
+		'items_list_navigation'      => __( 'Type list navigation', 'wporg-learn' ),
+	);
+
+	$args = array(
+		'labels'            => $labels,
+		'hierarchical'      => true,
```
Comment (tellyworth) [inline wp-content/plugins/wporg-learn/inc/taxonomy.php:462]: Is this intended to be hierarchical?

File: wp-content/plugins/wporg-learn/inc/taxonomy.php [hunk 2]
```diff
@@ -429,6 +430,51 @@ function register_workshop_topic() {
...
+		'menu_name'                  => __( 'Types', 'wporg-learn' ),
+		'all_items'                  => __( 'All types', 'wporg-learn' ),
+		'parent_item'                => __( 'Parent type', 'wporg-learn' ),
+		'parent_item_colon'          => __( 'Parent type:', 'wporg-learn' ),
+		'new_item_name'              => __( 'New Type Name', 'wporg-learn' ),
+		'add_new_item'               => __( 'Add Type', 'wporg-learn' ),
+		'edit_item'                  => __( 'Edit Type', 'wporg-learn' ),
+		'update_item'                => __( 'Update Type', 'wporg-learn' ),
+		'view_item'                  => __( 'View Type', 'wporg-learn' ),
+		'separate_items_with_commas' => __( 'Separate Type with commas', 'wporg-learn' ),
```
Comment (tellyworth) [inline wp-content/plugins/wporg-learn/inc/taxonomy.php]: Should this be plural? "Separate Types with commas"?

**Review by tellyworth — COMMENTED** [review]
Happy to merge this if the assumptions are correct.

File: wp-content/plugins/wporg-learn/inc/taxonomy.php (hunk 1 above)
Comment (hlashbrooke) [reply wp-content/plugins/wporg-learn/inc/taxonomy.php:462]: Yes - that's the intention here.

File: wp-content/plugins/wporg-learn/inc/taxonomy.php (hunk 2 above)
Comment (hlashbrooke) [reply wp-content/plugins/wporg-learn/inc/taxonomy.php]: Good catch - will update.

**Pull request merged**